pegasus/sites.v3/code.org/public/privacy-notice.md

```diff
@@ -7,7 +7,7 @@ title: Privacy Notice
 Date of Last Revision: August 24, 2016
```

Comment:
Does the date of last revision get updated automatically or does it need to be updated here?